### DISCUSSION TITLE
Expansion table in quasar

### DISCUSSION THREAD

**Discussion by nishantagrahari** [quasarframework/quasar#16174] [CLI - PWA mode]
Hey can any one help me to implement multi level  ( min 3) expansion table in quasar

**Comment by metalsadman** [comment]
you can take the examples from the docs https://quasar.dev/vue-components/table#expanding-rows

**Reply by nishantagrahari** [reply]
Yeah i tried referring from website but feeling difficult to implement multilevel row expansion

**Reply by metalsadman** [reply]
make a codepen of what you've tried.
